package/CHANGELOG

```diff
@@ -19,6 +19,7 @@ The rules for this file:
  * 2.4.0
 
 Fixes
+  * LAMMPS Dump Reader translates the box to the origin (#3741)
```

Comment:
Make all the uses of LAMMPSDump the same, `LAMMPSDump`.

currently 3 different versions